recipes/polars/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+export RUSTFLAGS='-C target-feature=+fxsr,+sse,+sse2,+sse3,+ssse3,+sse4.1,+sse4.2,+popcnt,+avx,+fma'
```

Comment:
Why?

Reply:
You can see the initial comment here: 
https://github.com/conda-forge/staged-recipes/pull/18906#pullrequestreview-971773615